tests/e2e/env/e2e.sh

```diff
@@ -25,6 +25,14 @@ for arg in "$@"; do
 
 done
 
+# Remove the latest trailing slash from the URL
+base_url=$(echo "$base_url" | sed 's:/*$::')
+
+# Helper to determine if we're running against Jurassic Ninja
+if [[ $base_url == *".jurassic.ninja" ]]; then
+  test_env="$test_env IS_JURASSIC='1'"
```

Comment:
> `IS_JURASSIC`

What is this used for? Should we call it a flag instead of a helper?

🤏 Also, this line specifically is indented with 2 spaces, the whole file is indented with spaces while the other `.sh` files are using tabs.

Reply:
I'll use this to get the SSH credentials for the JN site from the `wp-admin`, perhaps in another PR that separates the setup script from the tests.

Reply:
https://github.com/woocommerce/woocommerce-gateway-stripe/pull/2522/commits/d08282004ddcbc2cf70c68bc0e497c87615425fe